packages/editor/src/components/document-outline/index.js

```diff
@@ -64,7 +64,7 @@ const computeOutlineHeadings = ( blocks = [], path = [] ) => {
 
 const isEmptyHeading = ( heading ) => ! heading.attributes.content || heading.attributes.content.length === 0;
 
-export const DocumentOutline = ( { blocks = [], title, onSelect, isTitleSupported } ) => {
+export const DocumentOutline = ( { blocks = [], title, close, isTitleSupported } ) => {
```

Comment:
Why this prop was changed. What does a `close` prop mean in the context of a  `DocumentOutline` component.

Reply:
this is passing through the close functionality so we can close the outline when links are clicked, see https://github.com/WordPress/gutenberg/pull/10815/files#diff-486ea059bfd8ab47600431d73ad9824dR89 - is your question around the naming, eg should this be better named to reflect the functionality - eg called `onClick` or `closeOutline`?

Reply:
I think the question is specifically: What was wrong with `onSelect` that it required changing?

There's a bit of conconvention in the naming introduced here, much more imperative "close" than event-driven "onClose" that had existed previously / exists elsewhere.

Reply:
ah, ok - thanks for clarifying. Still getting used to the conventions here, I'll change that back to `onSelect`